server/config.py

```diff
@@ -1,3 +1,15 @@
 import os
+from dotenv import load_dotenv
+
+load_dotenv() #load all the environment variables in the .env file
 
 TEAM_NAME = os.environ['TEAM_NAME']
+POSTGRES_USER = os.environ['POSTGRES_USER']
+POSTGRES_PW = os.environ['POSTGRES_PW']
+POSTGRES_URL = os.environ['POSTGRES_URL']
+POSTGRES_DB = os.environ['POSTGRES_DB']
+
+
+
+
+
```

Comment:
Let's be consistent with formatting throughout and end files with a single empty line